katas/content/single_qubit_gates/prepare_arbitrary_state/index.md

```diff
@@ -0,0 +1,12 @@
+# Preparing an arbitrary state
```

Comment:
With the latest changes to infra, the title is included in the macro so it's better not to have it in the description of the exercise.